src/backends/sentry_backend_crashpad.cpp

```diff
@@ -183,6 +191,7 @@ sentry__crashpad_handler(int signum, siginfo_t *info, ucontext_t *user_context)
         } else {
             SENTRY_TRACE("event was discarded");
         }
+        sentry_value_decref(event);
```

Comment:
You're right, but wouldn't the preferable option be to only `decref` the `event` in the "discarded" path and leave the reference count given that we only transfer ownership from one ref to the other (and `event` isn't accessed any longer)? Or do you mean in case someone access `event` later on?

Reply:
This is all very academic, since you are in the crash path here, so you will not ever call into this function again. Though I believe its still worth doing

Reply:
Yeah I understand. Even if the effect of fixing this is inconsequential to the execution of the crash-handler, it should still be correct as documentation for future code additions. I just got rid of the local `event` and only use `data->crash_event` in the crash-handler (which is now defensifely `decref`ed).